library/src/main/java/com/novoda/simplechromecustomtabs/SimpleChromeCustomTabs.java

```diff
@@ -20,42 +19,35 @@
 
 public final class SimpleChromeCustomTabs implements WebNavigator, Connection, AvailableAppProvider {
 
-    private static Context applicationContext;
     private Connection connection;
     private WebNavigator webNavigator;
     private AvailableAppProvider availableAppProvider;
 
-    private SimpleChromeCustomTabs() {
-        //no-op
-    }
-
-    private static class LazyHolder {
-        private static final SimpleChromeCustomTabs INSTANCE = new SimpleChromeCustomTabs();
+    /**
+     * @deprecated There is no need to call it anymore in order to use this library,
```

Comment:
thats the debate though, because if we deprecate it, then in the future we add code back inside it, we would have to undeprecate it and you can't guarantee older versions of use where calling this method .. therefore adding the deprecated javadoc is also a future facing public api breaking change